stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -578,7 +578,8 @@ extension ProcessTerminationStatus {
       return CInt(signal) == SIGILL || CInt(signal) == SIGTRAP
     default:
       // This default case is needed for standard library builds where
-      // resilience is enabled
+      // resilience is enabled.
+      // FIXME: Need a way to suppress when not.
       return false
```

Comment:
There isn't, and I started a discussion on the list and opened [an SR](https://bugs.swift.org/browse/SR-3278).  One thing I'm thinking of is suppressing this warning when the enum in question is defined in the module the switch is in.